lib/ui/index.ts

```diff
@@ -155,20 +143,20 @@ const uiDeprecatedTag = (message, log = true) => {
   } else {
     return result;
   }
-};
+}
 
-const uiCommandDisabledBanner = (
-  command,
-  url = undefined,
-  message = undefined
-) => {
+export function uiCommandDisabledBanner(
+  command: string,
+  url: string | undefined = undefined,
```

Comment:
No need to default to `undefined`. Can just do `url?:string` and `message?: string` here. There are a few other places in this file where this is done so check those too

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1233/commits/3ec8a544996c20de5a030ed8db00991e855fe90d